src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -80,7 +80,8 @@ foam.CLASS({
         this.stack.push({
           class: 'foam.comics.v2.DAOCreateView',
           data: this.data.of.create({}, this),
-          config$: this.config$
+          config$: this.config$,
+          of: this.data.of
```

Comment:
Do you need both `of` and `data`? It feels redundant to have both. If the DAOCreateView knows the `of`, it could just create the `data` for you. And the `of` can be found from the `config` so you may only need the config and nothing more. What do you think?

Reply:
Remembered I was facing a weird bug before where the config of wasn't working, gunna retest to see if it works now

Reply:
It's chill we don't even use of in DAOCreateView because we crated it in data my bad

Reply:
Ah nvm, we needed to pass in an **of** property otherwise the facet won't trigger and render the AccountDAOCreateView

Reply:
Oh good call I overlooked that. Ok let's roll with this.